static/templates/stream_privacy.hbs

```diff
@@ -4,5 +4,5 @@
 {{else if is_web_public}}
 <i class="zulip-icon zulip-icon-globe" aria-hidden="true"></i>
 {{else}}
-<span class="hashtag"></span>
+<i class="hashtag"></i>
```

Comment:
Why make this change?
We use a `#` here, so `<i>` isn't needed.

Reply:
> Why make this change? We use a `#` here, so `<i>` isn't needed.

@ganpa3 I have updated it. Could you please review again?